build/generate-manifests.sh

```diff
@@ -40,22 +40,18 @@ fi
 mkdir -p "$BUNDLE_DIR"
 
 function checkDependencies() {
-  if ! [ -x "$(command -v jq)" ]; then
-    echo "Please install jq package.'"
-    exit 1
-  fi
-
-  # Looks for yq-python (https://github.com/kislyuk/yq) first, as installed in CI image, fallbacks to yq.
-  # Note there's another yq package, written in golang, which is not the one we want: https://github.com/mikefarah/yq
-  YQ=$(which yq-python 2>/dev/null || which yq 2>/dev/null || echo "")
-
+  # Looks for go-based yq (https://github.com/mikefarah/yq) first, as installed in CI image.
+  # Note there's another yq package, written in python, which is not the one we want: https://github.com/kislyuk/yq
+  YQ=$(which yq-go 2>/dev/null || which yq 2>/dev/null || echo "")
+  echo "Using yq: ${YQ}"
   if ! [ -x "$(command -v ${YQ})" ]; then
-    echo "Please install the python yq package, e.g. 'pip install --user yq'"
+    echo "Please install the go-based yq package (https://github.com/mikefarah/yq/releases/tag/v4.6.0)"
```

Comment:
Can we just add this as a tool and invoke it using `go run`?

Reply:
Best idea ever! Done!